frontend/packages/dev-console/src/components/projects/details/ProjectDetailsPage.tsx

```diff
@@ -1,33 +1,34 @@
 import * as React from 'react';
-import { Redirect } from 'react-router';
+import { match } from 'react-router';
 import { history } from '@console/internal/components/utils';
 import { ALL_NAMESPACES_KEY } from '@console/shared';
-import { ProjectsDetailsPage } from '@console/internal/components/namespace';
+import { NamespaceDetails, projectMenuActions } from '@console/internal/components/namespace';
 import { ProjectModel } from '@console/internal/models';
+import { DetailsPage } from '@console/internal/components/factory';
+import { ProjectDashboard } from '@console/internal/components/dashboard/project-dashboard/project-dashboard';
+import { withStartGuide } from '@console/internal/components/start-guide';
 import NamespacedPage, { NamespacedPageVariants } from '../../NamespacedPage';
-import {
-  PROJECT_LIST_URI,
-  redirectURI,
-  useActiveNamespace,
-  UseActiveNamespaceProps,
-} from './utils';
+import { useActiveNamespace, UseActiveNamespaceProps } from './utils';
+import ProjectAccessPage from '../../project-access/ProjectAccessPage';
+import CreateProjectListPage from '../CreateProjectListPage';
+import { handleProjectCreate } from '../../AddPage';
+
+export const MONITORING_ALL_NS_PAGE_URI = '/project-details/all-namespaces';
 
 const handleNamespaceChange = (newNamespace: string): void => {
   if (newNamespace === ALL_NAMESPACES_KEY) {
-    return;
+    history.push(MONITORING_ALL_NS_PAGE_URI);
   }
-
-  history.push(redirectURI(newNamespace));
 };
 
-export const ProjectDetailsPage: React.FC<UseActiveNamespaceProps> = ({
+export const ProjectDetailsPage: React.FC<UseActiveNamespaceProps & { match: match<any> }> = ({
   activeNamespace,
   ...props
 }) => {
   const allNamespaces = activeNamespace === ALL_NAMESPACES_KEY;
 
   if (allNamespaces) {
-    return <Redirect to={PROJECT_LIST_URI} />;
+    return <CreateProjectListPage onCreate={handleProjectCreate} title="Add" />;
```

Comment:
I don't think this is reachable code. There is another route in the plugin that leads to `AllProjectsDetailList`. Actually there are two routes.... why not just one?

Anyways. I think this code should be removed. Furthermore why would this redirect to the Add page? Currently we redirect back to the details of the newly created project